libs/full/actions_base/tests/performance/reflect_action_overhead.cpp

```
@@ -0,0 +1,179 @@
+    /// Returns a value that depends on the argument to prevent dead-code
+    /// elimination.
+    std::uint64_t identity(std::uint64_t n)
+    {
+        return n + 1;
+    }
+
+}    // namespace bench
+
+///////////////////////////////////////////////////////////////////////////////
+// Traditional approach: explicit action type + registration
+HPX_PLAIN_ACTION(bench::identity, bench_identity_action)
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If HPX_PLAIN_ACTION is a macro then please configure it.

(unknownMacro)